prow/plugins/verify-owners/verify-owners.go

```diff
@@ -101,7 +103,7 @@ type commentPruner interface {
 	PruneComments(shouldPrune func(github.IssueComment) bool)
 }
 
-type state struct {
+type info struct {
```

Comment:
JFYI this is the nomenclature used by the `approve` plugin as well:

https://github.com/kubernetes/test-infra/blob/30315db4aee52805b64dd3078df1441764d4f1aa/prow/plugins/approve/approve.go#L82

but I didn't want to touch that in this PR because that's used in lots of places in there and I'm not sure if it's worth it to touch that. Also, since the `state` struct in the approve plugin contains the current "state"/list of assignees, it reads better... :woman_shrugging: